### ISSUE TITLE
Trino to Spark conversion is broken

### ISSUE THREAD

**Issue by rnatarajan** [linkedin/coral#371]
Using Coral, I am able to convert the query with functions from [StaticHiveFunctionRegistry](https://github.com/linkedin/coral/blob/master/coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/functions/StaticHiveFunctionRegistry.java)  from Hive to Spark or Hive to Trino.

However the same is broken for Trino to Spark.

Steps for Reproduction using Coral Service

```
#Create Database and Table
CREATE DATABASE db
CREATE table db.tbl ( a INT)

#Convert Query 
SELECT count(*) from db.tbl

```

Error Message
`
At line 0, column 0: Unknown identifier '*'
`

**Comment by wmoustafa** [comment]
Yes, Trino to Spark is WIP. We will be publishing the next steps to complete the integration soon.

**Comment by rnatarajan** [comment]
Thanks and appreciate your quick response.

**Comment by Neelesh2512** [comment]
Hi, Any progress here? We want to try Trino to Spark in production usecases
